content/django-vs-flask.md

```diff
@@ -0,0 +1,100 @@
+# Django vs Flask: Which one should you choose for your project?
+[Django](https://www.djangoproject.com/) and [Flask](https://flask.palletsprojects.com/en/1.1.x/) are among the most popular Python web frameworks.
+In this article, we're going to review their main strengths and weaknesses to help you decide which one is a better fit for your project.
+
+
+# Django
+[![django-logo](images/flask-vs-django-django-logo.png)](https://www.djangoproject.com/)
+## Approach
+Django opts for an all batteries-included approach.
+From a powerful ORM to an authentication system or a templating engine, Django takes care of it all without the need to learn any extra libraries.
+
+It also has the concept of pluggable apps, which means you can drop an app folder inside your project and get its functionality right away with very little configuration. 
+
+## Main Features
+- Django has a very powerful ORM that exposes a lot of SQL to the python programming language.
+- It has Authentication and Authorization built-in, without the need for any extra configuration.
+- It also boasts a built-in admin site, and
+- a built-in mailing system.
+
+## Pitfalls
+- Being tightly coupled with its toolset, Django can be difficult to use in combination with other tools, like replacing the form library, for example.
+- While Django's **default scaffolding function** is not bad for small projects, it does not scale without modification when trying to manage things such as multiple environments.
+- This functionality implements the Model View Template pattern but requires some effort to implement other architecture types with it.
+
+## Developer Experience
+One of the main perks of using an all batteries included approach, is that the developer experience will be more or less the same across the framework.
+This proves very beneficial to people new to the framework, as they can follow a pattern and a guide to follow good practices by looking at existing code.
+Another benefit you get from this is that the core parts of every site are tried and tested.
+This is a very desirable feature for parts like authentication and authorization, or an admin panel used to provide easy back office access to your team.
+
+## Team size and structure
+Django is at its best for a small to medium team that plans to handle a middle to large sized code base
+The project structure takes that into account and file conflicts will be minimal.
+Since it has some sort of scaffolded default structure for it's projects is very easy to ramp up newbie developers into it with a senior supervising the team.
```

Comment:
```suggestion
Since it has some sort of scaffolded default structure for its projects, it's very easy to ramp up newbie developers into using it, with a senior developer supervising the team.
```